packages/main/src/themes/base/PopupBlockLayer-parameters.css

```diff
@@ -1,3 +1,3 @@
 :root {
-	--_ui5_popup_block_layer_background: color-mix(in oklch, transparent, var(--sapBlockLayer_Background) 60%);
+	--_ui5_popup_block_layer_background: color-mix(in oklch, transparent, var(--sapBlockLayer_Background) 20%);
```

Comment:
Opacity should be different in the different themes.
Do you have idea why `color-mix` is used here? Can't we just set `opacity`?